README.md

```diff
@@ -285,32 +285,62 @@ If the Ethereum endpoint is on mainnet, then it will be StarkNet Mainnet.
 
 ## Running with Docker
 
-The `pathfinder` node can be run in the provided Docker image.
-Docker image is the easiest way which does not involve a lot of python setup.
 The following assumes you have [docker installed](https://docs.docker.com/get-docker/) and ready to go.
 
-The example uses `$HOME/pathfinder` as the volume directory where persistent files used by `pathfinder` will be stored.
-It is easiest to create the volume directory as the user who is running the docker command.
-If the directory gets created by docker upon startup, it might be unusable for creating files.
+### Docker compose
+
+The [docker-compose.yml](./docker-compose.yml) file embeds everything to run or deploy goerli and mainnet nodes in one command.
+It uses env variables defined in the `.env` that you need to create from the `example.env` and populate with Ethereum RPC endpoints.
+
+```bash
+cp example.env .env
+# replace the value(s) of PATHFINDER_ETHEREUM_API_URL by the HTTP URL(s) pointing to your Ethereum node's endpoint
+```
+
+By default, `docker compose up` will run both a mainnet and a goerli node. If you want to run only one service, you can specify it after the `up`:
+
+```bash
+docker compose up # run both mainnet and goerli
+docker compose up starknet-mainnet # run mainnet only
+docker compose up starknet-goerli # run goerli only
+```
+
+To check if it's running well use `docker-compose logs -f`.
+
+The mainnet node runs on port 9546 while the goerli one runs on port 9545. You can check this by calling the `starknet_chainId` method:
 
 ```bash
-# ensure the directory has been created before invoking docker
-mkdir -p $HOME/pathfinder
-docker run \
-  --rm \
-  -p 9545:9545 \
-  --user "$(id -u):$(id -g)" \
-  -e RUST_LOG=info \
-  -e PATHFINDER_ETHEREUM_API_URL="https://goerli.infura.io/v3/<project-id>" \
-  -v $HOME/pathfinder:/usr/share/pathfinder/data \
-  eqlabs/pathfinder
```

Comment:
Please don't remove the parts on how to run pathfinder without `docker-compose`.

Reply:
ok I leave it there. May I ask the benefit of this? I guess that it's easier to write `docker compose up starknet-mainnet` than to copy/paste those lines

Reply:
as well as not using a `docker volume` but binding a local folder?